contracts/bonding/BondingManager.sol

```diff
@@ -153,7 +153,7 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
     }
 
     /**
-     * @dev Set number of active transcoders. Only callable by Controller owner
+     * @notice Set number of maximum active transcoders. Only callable by Controller owner
```

Comment:
"Set maximum number of active transcoders"

Reply:
fixed in b79bb67